pkg/server/server.go

```diff
@@ -417,18 +417,20 @@ func (s *Server) Run(ctx context.Context) error {
 
 		klog.Infof("Finished starting APIExport and APIBinding informers")
 
-		// bootstrap root workspace phase 0, no APIBinding resources yet
-		if err := configrootphase0.Bootstrap(goContext(ctx),
-			kcpClusterClient.Cluster(tenancyv1alpha1.RootCluster),
-			apiextensionsClusterClient.Cluster(tenancyv1alpha1.RootCluster).Discovery(),
-			dynamicClusterClient.Cluster(tenancyv1alpha1.RootCluster),
-		); err != nil {
-			// nolint:nilerr
-			return nil // don't klog.Fatal. This only happens when context is cancelled.
+		if s.options.Extra.ShardName == tenancyv1alpha1.RootCluster.String() {
```

Comment:
nit: should we introduce a RootShard constant? This is not really the cluster.

Reply:
why not

Reply:
done.